README.md

```diff
@@ -42,19 +42,32 @@ on first run, this will be populated with a basic config if it doesn't exist.
 The default config is:
 
 ```toml
-# How often to check battery status, in seconds.
-interval_secs = 30
+# How often to check battery status, in milliseconds.
+interval = 30000
 
-# At what percentage of battery capacity to notify about low battery.
+# At what percentage of battery capacity to notify about low battery, set to 0 to disable.
```

Comment:
Considering #73, instead of making low_pct/sleep_pct more nuanced, let's make it generic:

1. Have an enum that represents all notification states
2. Allow the user to define what happens when those states are triggered (can be configured as a table in the TOML)
3. Refuse to run if we see a state that's not known

Reply:
Like provide full granularity of enabling/disabling every notification type we send? Considering every battery state as a separate notification type? I am not sure I completely understand what you said.

Reply:
1. ~~Have an enum that represents all notification states~~
1. ~~Allow the user to define what happens when those states are triggered (can be configured as a table in the TOML)~~
1. Refuse to run if we see a state that's not known

Reply:
This was marked as resolved, but it doesn't use the enum, right?

Reply:
It uses a struct now, please check the latest commit. I think only the refusing to run part is left now.